librz/bin/p/bin_mach0.c

```diff
@@ -408,9 +408,9 @@ static RzPVector /*<RzBinReloc *>*/ *relocs(RzBinFile *bf) {
 				free(ptr);
 				break;
 			}
-			ptr->import = imp;
+			ptr->import = rz_bin_import_clone(imp);
 		} else if (reloc->ord >= 0 && reloc->ord < rz_pvector_len(&bin->imports_by_ord)) {
-			ptr->import = rz_pvector_at(&bin->imports_by_ord, reloc->ord);
+			ptr->import = rz_bin_import_clone(rz_pvector_at(&bin->imports_by_ord, reloc->ord));
```

Comment:
even tho this is likely ok, do first an assignment to `RzBinImport *imp;` and then clone to `ptr->import`